File: source/SEP3 - tier 2 (IntelliJ)/src/main/java/org/EvaAndTheSovietHouseholdAppliances/model/UserPw.java
package org.EvaAndTheSovietHouseholdAppliances.model;

import java.util.Objects;

/**this class serves as a Data Object for User-Password*/
public class UserPw
{

    /**name of the user*/
    public String Name;
    /**password of the user*/
    public String Password;

    /**initialises all instance variables*/
    public UserPw(String name, String password)
    {
        this.Name = name;
        this.Password = password;
    }

    public String toString()
    {
        return Name + " " + Password;
    }

    public boolean equals(Object obj)
    {
        if (!(obj instanceof UserPw))
            return false;
        return (Objects.equals(((UserPw) obj).Name, Name) && Objects.equals(((UserPw) obj).Password, Password));
    }
}

File: source/SEP3 - tier 1 (Rider)/Model/UserPw.cs
namespace WebApplication7.Model
{
    public class UserPw
    {
        public string Name { set; get;}
        public string Password { set; get; }

        public UserPw(string name, string password)
        {
            Name = name;
            Password = password;
        }
        
        public override string ToString()
        {
            return Name + " " + Password;
        }
    }
}